src/main/scala/com/rbmhtechnology/eventuate/Recovery.scala

```diff
@@ -167,23 +195,34 @@ private object Acceptor {
   case object Process
   case class Recover(links: Set[RecoveryLink], promise: Promise[Unit])
   case class RecoveryStepCompleted(link: RecoveryLink)
-  case object RecoveryCompleted
+  case object MetadataRecoveryCompleted
+  case object EventRecoveryCompleted
 }
 
 /**
- * Manages [[RecoveryActor]]s to execute the disaster recovery steps for each replication link in `links`
+ * Recovery of an [[ReplicationEndpoint]] is a two steps process:
+ *
+ * - Metadata recovery: A [[ReplicationReadEnvelope]] from remote is responded with
+ *   [[ReplicationReadSuccess]] with the current sequence nr to update remote's replication progress
+ * - Event recovery: [[ReplicationReadResponseEnvelope]] responses are intercepted to detect when all _lost_ events
```

Comment:
... when all events, known to exist remotely at the beginning of recovery, are replicated ...